guides/manual-installation.md

```diff
@@ -469,6 +469,69 @@ Expected output if Trento web/wanda is ready and the database connection is setu
 {"ready":true}{"database":"pass"}
 ```
 
+## Integrations
+
+### OpenID Connect
+
+Trento integrates with an identity provider (IDP) that uses the OpenID Connect (OIDC) protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
+
+
+#### User Roles and Authentication
+
+User authentication is entirely managed by the IDP, which is responsible for maintaining user accounts. 
+A user, who does not exist on the IDP, is unable to access the Trento web console.
+If the default admin user created during the installation process matches the authenticated user’s IDP username, that user is automatically granted `all:all` permissions within Trento.
+Create a new user on IDP and login to Trento to finalize the user creation on Trento.
```

Comment:
I don't think we should have this paragraph (from here to `Enabling OIDC` actually). In a IDP environment, they should know what users they have and how to manage them. We cannot ask them to create users.
They should only know that they must put at least one user matching the `ADMIN_USER` so they are able to change abilities for other authenticated user.